asmcli/commands/help.sh

```diff
@@ -290,6 +293,7 @@ FLAGS:
 
      --print_config
      --disable_canonical_service
+  --managed_certificates
```

Comment:
I think this is three spaces too far to the left and doesn't align with the other flags

Reply:
fixed indentation.